packages/perseus/src/widget-container.jsx

```diff
@@ -68,6 +68,10 @@ class WidgetContainer extends React.Component<Props, State> {
             "perseus-widget-container": true,
             "widget-highlight": this.props.shouldHighlight,
             "widget-nohighlight": !this.props.shouldHighlight,
+            // HACK(matthewc): perseus-widget-container is setting a font-size
+            // but we want the definition prompt to match the surrounding font
+            // I'm sorry, but there's a time crunch
+            "perseus-widget__definition": this.props.type === "definition",
```

Comment:
@jeremywiebe We discussed giving every widget a unique class name, but I decided to just do the definition widget since doing it for all widgets changed a ton of snapshots.

Happy to go back to what we discussed, but thought this would be a little more explicit/surgical.

Reply:
Let's go with your approach and be surgical for now! :)